.github/workflows/ci-base-tests.yml

```diff
@@ -21,15 +19,12 @@ jobs:
             -logfile ./xdummy.log \
             -config /etc/X11/xorg.conf :1 &
           cd $GITHUB_WORKSPACE
-          python3.7 -m venv "${venv_dir}"
-          . "${venv_dir}/bin/activate"
           pip install --upgrade pip
           pip install wheel
-          pip install -r requirements.txt
+          pip install --no-deps -r requirements.txt
```

Comment:
why?

if they're cached it'll just go over em quickly

Reply:
I was getting dependency collisions.

Reply:
This is likely _not_ the solution to that. If these are dep collisions of our own dependencies than we need to ease up our requirements.txt / setup.py to let them resolve.